### ISSUE TITLE
Fix the "Contact Us" form

### ISSUE THREAD

**Issue by cafelow** [rtcharity/resources.eahub.org#648]
- [x] I'm not sure where it is going now.. a black hole perhaps.  I'd like it to go to catherine@eahub.org

- [ ]  Can text be changed to 
Large font: "Contact the EA Hub Resources Team"
Small font: "The EA Hub Resources are a work in progress, and had been created by a team of EAs around the world, most of whom are volunteers. We'd love to have your feedback to improve the site. Please suggest any changes in the form below, or if you have substantial suggestions please ask and we can send you the google drive folders for you to review - we would love to have your help!

**Comment by sebgrebe** [comment]
@cafelow  The form is now wired up properly.
Regarding the text changes: Changed the header. I find your small font suggestion a bit too wordy. Can we cut this down a bit?

**Comment by cafelow** [comment]
Thanks! How about this:
"We would love to have your feedback! Please fill in the form below or email Catherine on catherine@eahub.org.  If you have substantial suggestions we can send you the google drive folders for you to review."

**Comment by cafelow** [comment]
Thanks! How about this:
"We would love to have your feedback! Please fill in the form below or email Catherine on catherine@eahub.org. If you have substantial suggestions please suggest and comment on the [Hub Resources google docs](https://docs.google.com/document/d/18S2AG9RgFQD9GTCBbBcFVrgZNh7UuMsayTp--jDLDqY/edit#)."

**Comment by sebgrebe** [comment]
I don't think that we should offer two different ways of contacting here, that's confusing. If we should see that the form is used a lot and we might reach our 100/month limit, we can think again

**Comment by cafelow** [comment]
Thanks for adding that info. I do have to disagree with you about the email though... I think it is important we give an email address as some people (including me) greatly prefer email to filling in contact forms (trackability, less worry about your request getting there, adding attachments, less of a feeling that we are being anonymous). 

If you feel strongly that we should only have one method of contact, then I'd prefer an email address over the contact form.

**Comment by sebgrebe** [comment]
@cafelow OK, added your email address. I think in the long run it would be better if we could have a generic email address, like "resources@eahub.org".
